haiku-apps/xrs/xrs-1.9.1.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A simple rhythm station for creating music loops"
+DESCRIPTION="XRS is a simple Rhythm Station.\
```

Comment:
Sorry for nit-picking :) No trailing slash needed here:
```diff
- DESCRIPTION="XRS is a simple Rhythm Station.\
+ DESCRIPTION="XRS is a simple Rhythm Station.
```

Reply:
Although the line if fine it would look like: 
```
XRS is a simple Rhythm Station.You can use it to create music compositions and loops of your preferred elettronic music genres.
```
 (see the missing space?)
If you leave out the trailing slash it would look like:
```
XRS is a simple Rhythm Station.
You can use it to create music compositions and loops of your preferred elettronic music genres.
```

Reply:
Thank you for the clarification, crystal clear!